src/containers/coursePageDetails/index.jsx

```diff
@@ -42,106 +70,118 @@ class coursePageDetails extends Component {
       profits,
       courseReviews,
     } = this.state.course;
-    const { isLoading, showReviewsNum, error } = this.state;
+
+    const { isLoading, showReviewsNum, error, subscribed } = this.state;
+
+    const { isAuthenticated } = this.props;
     const { t: translate } = this.props;
 
     return (
       <main className={styles.coursePageDetailsWrapper}>
-        {isLoading ? (
+        {isLoading && (
           <Spinner />
-        ) : error ? (
+        )}
+        {error ? (
           <div className={styles.error}>{error.message}</div>
         ) : (
-          <div>
-            <h2 className={styles.courseTitle}>{courseName}</h2>
-            <p className={styles.shortDescription}>{description}</p>
-
-            <div className={styles.courseStats}>
-              <div className={styles.stars}>
-                <StarRatings
-                  rating={rating}
-                  starRatedColor="#ff8c05"
-                  starEmptyColor="#515151"
-                  starDimension="24px"
-                  starSpacing="3px"
-                  numberOfStars={5}
-                  name="courseRating"
-                />
-              </div>
-
-              <span>{rating}</span>
-
-              <span className={styles.verticalLine} />
-
-              <span>
-                <font>{translate('Students')}: </font>
-                {numberOfEnrolledStudents}
-              </span>
-            </div>
-
             <div>
-              <span>
-                <font>{translate('Author')}: </font>
-                {authors}
-              </span>
-
-              <span className={styles.verticalLine} />
-
-              <span>
-                <font>{translate('Language')}: </font>
-                {language}
-              </span>
-            </div>
-
-            <hr />
-            <p className={styles.profitTitle}>{translate('Your profits')}</p>
-
-            <ul className={styles.profitList}>
-              {profits.map(profit => (
-                <li key={profit.id}>{profit.description}</li>
-              ))}
-            </ul>
-
-            <hr />
-            <p className={styles.reviewsTitle}>{translate('Reviews')}</p>
-            {courseReviews.slice(0, showReviewsNum).map((review, key) => (
-              <div className={styles.reviewWrapper} key={key}>
-                <div>
-                  <span>{review.user.userName}</span>
-
-                  <div className={styles.stars}>
-                    <StarRatings
-                      rating={review.rating}
-                      starRatedColor="#ff8c05"
-                      starEmptyColor="#515151"
-                      starDimension="24px"
-                      starSpacing="3px"
-                      numberOfStars={5}
-                      name={`review${review.key}`}
-                    />
+              <header className={styles.headerWrapper}>
```

Comment:
Maybe it's better to use simple `section` instead of `header` tag?

Reply:
It's a header content, isn't it?

Section contents header tag. It's better to wrap all components to the section tag. But I don't want to do that, because of breaking the working code.